### PR TITLE
Implementing feature 'Skip rebuild'

### PR DESCRIPTION
**Pull request by unitorzero**
**skip Rebuild** is a flag in the file `entity Defs/{Entity Name}.json` for indicate Entity that do NOT need to be rebuild when CRM will rebuilds.
Possible cases:
  - the need to do rebuild for the entire CRM without touching individual Entities (tables);
  - creating a DB-side view for Entity content;

### CODE REVIEW

File: application/Espo/Core/Utils/Database/Schema/Converter.php [hunk 1]
```diff
@@ -186,6 +186,8 @@ public function process(array $ormMeta, $entityList = null)
         $tables = array();
         foreach ($ormMeta as $entityName => $entityParams) {
 
+            if ($entityParams['skipRebuild']) continue;
```
Comment (yurikuzn) [inline application/Espo/Core/Utils/Database/Schema/Converter.php]: `if ($entityParams['skipRebuild']) continue;` 
=>
`if ($entityParams['skipRebuild'] ?? false) continue;`

File: contributors/unitorzero.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+I hereby agree to the terms of the EspoCRM Contributor License Agreement v1.0.
+
+2020-03-17
+
+Peter Liubko
```
Comment (yurikuzn) [inline contributors/unitorzero.md]: This should be in another repository: https://github.com/espocrm/cla

File: application/Espo/Core/Utils/Database/Orm/Converter.php [hunk 3]
```diff
@@ -172,6 +172,8 @@ public function process()
                 continue;
             }
 
+            $ormMetadata[$entityName]['skipRebuild'] = $entityMetadata['skipRebuild'] ?? false;
```
Comment (yurikuzn) [inline application/Espo/Core/Utils/Database/Orm/Converter.php]: `if ($entityMetadata['skipRebuild'] ?? false) $ormMetadata[$entityName]['skipRebuild'] = true;`

**Comment by unitorzero** [comment]
I fixed issues and create the contributor file in [cla]( https://github.com/espocrm/cla) repository.

**Pull request merged**